app/api-v1/controllers/Recipe/index.js

```diff
@@ -51,7 +65,7 @@ module.exports = {
         ],
       }
       const token = await runProcess(payload, req.token)
-      const transaction = await client
+      const transaction = await db.client
         .from('recipe_transactions')
         .insert({
```

Comment:
need to add `type: 'Creation'`?

Reply:
and `status: 'submitted'` needs capital `Submitted` to match column enum

Reply:
I've a fix for this along with making runProcess fire and forget in IN-229 another PR. trying to keep this within IN-231 story. We have gone a little beyond during the pairing sessions yesterday.

Reply:
the link https://github.com/digicatapult/inteli-api/pull/28/files

Reply:
yeah this is frustrating. I can't work out at this point what's broken what. I really wish we hadn't merged #15 which has caused many of these issues. This mostly looks ok, but @n3op2 please don't work on new functionality until this is all cleaned up